document/js-api/index.bs

```diff
@@ -803,10 +813,10 @@ which can be simultaneously referenced by multiple {{Instance}} objects. Each
     The <dfn method for="Table">set(|index|, |value|)</dfn> method, when invoked, performs the following steps:
     1. Let |tableaddr| be **this**.\[[Table]].
     1. Let |values| be **this**.\[[Values]].
-    1. If |value| is null, let |ref| be [=ref.null=].
-    1. Otherwise,
-        1. If |value| does not have a \[[FunctionAddress]] internal slot, throw a {{TypeError}} exception.
-        1. Let |ref| be [=ref.func=] |value|.\[[FunctionAddress]].
+    1. Let (<var ignore>limits</var>, |elementType|) be [=table_type=](|tableaddr|).
+    1. If |value| is undefined and |elementType| is {{TableKind/anyfunc}},
```

Comment:
And here.

Reply:
This is less of an issue, because the argument is not optional, so it can't be "missing". OTOH, this means it clashes even more with https://www.w3.org/Bugs/Public/show_bug.cgi?id=23602. I think I'll make this optional as well.